.github/draft_release_notes.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+
```

Comment:
I am a fan of `set -euo pipefail` in scripts to smoke out unset variables and pipefails. Not sure about the PR list, which go through a couple of `grep`s, so maybe the `pipefail` could be omitted.

Reply:
I'll add it

Reply:
actually `set -euo pipefail` broke my processing of the return code from `git merge-base` @HeavyWombat 

so I'm going to refrain from adding that